osbenchmark/worker_coordinator/runner.py

```diff
@@ -677,29 +677,21 @@ class ForceMerge(Runner):
     """
 
     async def __call__(self, opensearch, params):
-        # pylint: disable=import-outside-toplevel
-        import opensearchpy
         max_num_segments = params.get("max-num-segments")
         mode = params.get("mode")
         merge_params = self._default_kw_params(params)
         if max_num_segments:
             merge_params["max_num_segments"] = max_num_segments
         if mode == "polling":
-            complete = False
-            try:
-                request_context_holder.on_client_request_start()
-                await opensearch.indices.forcemerge(**merge_params)
-                request_context_holder.on_client_request_end()
-                complete = True
-            except opensearchpy.ConnectionTimeout:
-                pass
-            while not complete:
-                await asyncio.sleep(params.get("poll-period"))
-                tasks = await opensearch.tasks.list(params={"actions": "indices:admin/forcemerge"})
-                if len(tasks["nodes"]) == 0:
-                    # empty nodes response indicates no tasks
+            merge_params["wait_for_completion"] = "false"
```

Comment:
Please return a diagnostic if `wait_for_completion` is provided in params and happens to be set to `true`.  This will revert to the blocking mode in any case.

Reply:
@gkamat Sorry i don't understand what do you mean return a diagnostic. Can you provide an example on how to do that?

Reply:
You can print a warning message using `console.warn()`.

Reply:
Ack